### PR TITLE
Add HelloWorld solution and tests

### PR DESCRIPTION
**Pull request by robkeim**
While this one's a low hanging fruit, I enjoyed it in the F# track as it helped me get accustomed to the work flow without having any difficult code to write.

### CODE REVIEW

**Comment by ErikSchierboom** [comment]
@robkeim Good idea! There is no thing left to do: the `hello-world` entry you added to the `exercises` array in the `config.json` file should become the first entry in that list.

**Comment by robkeim** [comment]
I wasn't sure what the rule is for exercise ordering, but agreed that it makes sense to have `hello-world` be the first one :)

I've gone ahead and updated the PR.

File: config.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "repository": "https://github.com/exercism/xcsharp",
   "active": true,
   "problems": [
+	"hello-world",
```
Comment (ErikSchierboom) [inline config.json]: The indentation is off, sorry :)

File: config.json [hunk 2]
```diff
@@ -59,10 +60,15 @@
     "simple-linked-list",
     "matrix",
     "queen-attack",
-    "ocr-numbers",
-	"hello-world"
+    "ocr-numbers"
   ],
   "exercises": [
+	{
```
Comment (ErikSchierboom) [inline config.json]: Also has the wrong indentation.

**Comment by ErikSchierboom** [comment]
@robkeim There are just two small indentation problems, after that it's perfect!

File: config.json (hunk 1 above)
Comment (robkeim) [inline config.json]: Ah, I got bit by my editor using tabs :)

**Comment by ErikSchierboom** [comment]
Merged! Thanks. :tada:

**Pull request merged**